doc/dbus.rst

```diff
@@ -129,13 +129,21 @@ known to ratbagd.
 
 .. attribute:: Capabilities
 
-        :type: au
+        :type: a(us)
         :flags: read-only, constant
 
 	The capabilities supported by this device. see
 	:cpp:enum:`ratbag_device_capability` in libratbag-enums.h for the
 	list of permissible capabilities.
 
+	The content of this property is a list of tuples with the
+	capability as first entry and a string specifying the modes as second
+	entry. Characters allowed in this string are 'r' for read-only, 'w'
```

Comment:
I read this as the capability being either `r` *or* `w`, not that `rw` is also a possiblity.